tools/val/val.cpp

```diff
@@ -63,10 +63,9 @@ int main(int argc, char** argv) {
     if ('-' == cur_arg[0]) {
       if (0 == strcmp(cur_arg, "--version")) {
         printf("%s\n", kBuildVersion);
-        printf("Targets:\n  %s\n  %s\n  %s\n",
-               spvTargetEnvDescription(SPV_ENV_UNIVERSAL_1_0),
-               spvTargetEnvDescription(SPV_ENV_VULKAN_1_0),
-               spvTargetEnvDescription(SPV_ENV_UNIVERSAL_1_1));
+        printf("Targets:\n  %s\n  %s\n",
```

Comment:
I see.  Vulkan is supported via --vulkan. 
For another CL, I'd like to see a --target-env option to match what glslc does.  But that's another story.